### DISCUSSION TITLE
latest FW:  F7 not showing .prg files

### DISCUSSION THREAD

**Discussion by snikhaas** [frntc/SIDKick-pico#52] [General]
Hi,

First of all Thnx for the new FW release and the new feature for adding .prg files.

Added a few games .prg with SKpicoPatch ...
Games do start using sys command , but are not showing in the sidkick pico config pressing F7...

Is there something i need to change to make it work ?

Any insight how to make FM work ,, i dont see how to activate it

ps.  using FW : SKpico1_DAC_PWM_LED.uf2

**Comment by frntc** [comment]
Argh, I must have forgotten to uncomment some code :-) Will have a look soon.

**Reply by snikhaas** [reply]
Thanks for the support 👍🏻

**Reply by frntc** [reply]
should work now, plz have a look!

**Reply by snikhaas** [reply]
Yes,  Works like a charm ;-)

Thanks for the support....

Is there How To for the FM sound ,,, do i need to connect IO from the cartridge port to make it work ?

gr,
Marc

**Reply by frntc** [reply]
You have to connect the "A5" (or "A5/A6") pin to A6, and the "A8/IO" pin to pin 10 of the expansion port (the "IO2"-signal which represents the $df?? address range).

**Reply by snikhaas** [reply]
Thnx for the info ...👍🏻
